### ISSUE TITLE
 file_put_contents(G:\GitHub\smartfinance\resources\laravel-code-generator\sources\resources_map.json): failed to open stream: No such file or directory

### ISSUE THREAD

**Issue by itpocfinance** [CrestApps/laravel-code-generator#113]
### Before you submit an issue please read this
This repository is only for reporting bugs or issues. If you need support, please use Stack Overflow using **laravel-code-generator** tag (https://stackoverflow.com/questions/tagged/laravel-code-generator)

Please provide us with details by completing the following requirements. Issues with not enough info are likely to be closed without resolution.

### Environment:
- Laravel-Code-Generator Version: 2.3
- Laravel Version: 5.8

### Description:
I have used these two commands ...
composer require crestapps/laravel-code-generator --dev
php artisan vendor:publish --provider="CrestApps\CodeGenerator\CodeGeneratorServiceProvider" --tag=default.

but in resource folder laravel-code-generator folder not comming

**Comment by kenmdavids** [comment]
I had the same issue, but I solved it like this,
I first removed the package using
`composer remove crestapps/laravel-code-generator --dev`
or
`composer remove crestapps/laravel-code-generator`
and remove all the instances like
this
```
if ($this->app->runningInConsole()) {
    $this->app->register('CrestApps\CodeGenerator\CodeGeneratorServiceProvider');
}
```
then install this particular version
`composer require crestapps/laravel-code-generator "2.2.*"`
you will be good to go

**Comment by itpocfinance** [comment]
Thank you

**Comment by CrestApps** [comment]
This issue should be fixed as of v2.3.2.
